src/core/backend_context.cc

```diff
@@ -209,8 +200,9 @@ BackendContext::SetFixedSizeOutputBuffer(
 Status
 BackendContext::CopyBuffer(
     const std::string& name, const TRTSERVER_Memory_Type src_memory_type,
-    const TRTSERVER_Memory_Type dst_memory_type, const size_t byte_size,
-    const void* src, void* dst, bool* cuda_used)
+    const int src_memory_type_id, const TRTSERVER_Memory_Type dst_memory_type,
```

Comment:
Don't use "int" as the type of memory_type_id (and don't cast to int in the calls to this function). The the type of memory_type_id, which is int64_t.

Reply:
Fixed